### ISSUE TITLE
SharedInstance viewable packet issues

### ISSUE THREAD

**Issue by oHate** [Minestom/Minestom#549]
Viewable packets cause certain packets to not get sent to other players.

Minecraft Screenshot (Note: oHate joined before Eimch):
![unknown](https://user-images.githubusercontent.com/46694862/147378883-439754c0-c422-4a19-86de-4f2d85e8a8c5.png)

Code to reproduce this issue:
```java
InstanceContainer container = MinecraftServer.getInstanceManager().createInstanceContainer(new AnvilLoader("hub"));
SharedInstance instance = MinecraftServer.getInstanceManager().createSharedInstance(container);
Pos spawn = new Pos(0.5, 72, 0.5);

EventNode<PlayerEvent> node = EventNode.type("hub", EventFilter.PLAYER)
        .addListener(PlayerLoginEvent.class, event -> {
            event.setSpawningInstance(instance);
            event.getPlayer().setRespawnPoint(spawn);
});

MinecraftServer.getGlobalEventHandler().addChild(node);

**Comment by TheMode** [comment]
For reference, this is due to movement packets being sent to all the viewers of the current player chunk. But the chunk viewers only come from the instance container: https://github.com/Minestom/Minestom/blob/1.18/src/main/java/net/minestom/server/instance/Chunk.java#L68-L69

**Comment by TheMode** [comment]
It has finally been fixed by 52852d339a5239dc1878908ed114444ea8d6567e
